tests/test_config.py

```diff
@@ -92,83 +99,60 @@ def test_parse():
         assert actual["path"] == str(Path(config_path.parent / "./path").resolve())
 
 
+# TODO: remove this when SimulationConfig has manifest parsing
```

Comment:
is this fixed in libsonata?

Reply:
Yeah, this actually currently depends on when/if we can remove the Parser in config (as it is still used by circuit_validation)
-> removed comment